crates/glaredb/src/local.rs

```diff
@@ -73,25 +73,9 @@ impl LocalSession {
                     format!("Connected to remote GlareDB server: {}", u.cyan()),
                 )
             } else {
-                let tls_conf = if opts.disable_tls {
-                    None
-                } else {
-                    match (opts.ca_cert_path.clone(), opts.domain.clone()) {
-                        (Some(ca_cert_path), Some(domain)) => Some(TlsConfig {
-                            ca_cert_path,
-                            domain,
-                        }),
-                        _ => {
-                            return Err(anyhow!(
-                                "Specify ca-cert-path and domain in --args for TLS"
-                            ));
-                        }
-                    }
-                };
-
-                let mut url: ProxyDestination = url.try_into()?;
-                url = url.with_tls(tls_conf);
-                let client = RemoteClient::connect_with_proxy_destination(url).await?;
+                let client =
+                    RemoteClient::connect_with_proxy_destination(url.try_into()?, !opts.enable_tls)
```

Comment:
the `!` here is so that when we change the args from `enable` -> `disable`, we don't need to change further logic inside `connect_with_proxy_destination`

Reply:
I'm fine with this. We can change in the future.

Reply:
I'm now realizing I should've done this approach with the server side too...oops